turbo-txpool/src/tx.rs

```diff
@@ -332,6 +442,86 @@ mod tests {
             .into()
         );
 
+        let vtx = VerifiedTx::new(tx).unwrap();
+        let hash = H256::from([
+            0x20, 0x94, 0x66, 0xe8, 0xf6, 0xb9, 0x69, 0xc2, 0x07, 0x5b, 0xf5,
+            0x46, 0x93, 0x82, 0x65, 0xa1, 0x3e, 0xee, 0x23, 0xca, 0xf7, 0xd5,
+            0x9e, 0x4d, 0x7c, 0x4e, 0x54, 0xc2, 0x6e, 0xb4, 0x82, 0xec,
+        ]);
+        assert_eq!(vtx.hash, hash);
+
+        Ok(())
+    }
+
+    #[test]
+    fn decode_1000000() -> Result<(), Error> {
+        let txbytes = [
```

Comment:
I think I generally prefer hex in tests because it lets me reason about it in my head better. Not sure if you have different thoughts on this.